libnavigator/src/androidTest/res/raw/directions_response_two_leg_route.json

```diff
@@ -0,0 +1,6104 @@
+{
```

Comment:
```
➜  raw git:(kyv-nn-router-interface-) cat directions_response_two_leg_route.json | tr -d " \t\n\r" > directions_response_two_leg_route-flat.json
➜  raw git:(kyv-nn-router-interface-) ✗ ll
total 432
-rw-r--r--  1  staff    49K Mar  7 19:16 directions_response_two_leg_route-flat.json
-rw-r--r--@ 1  staff   162K Mar  7 19:13 directions_response_two_leg_route.json
````

File it a bit smaller if you cut it to a single line

Reply:
@kmadsen json locates in the test module and does not inflate SDK's size. I wouldn't change to not wait for all tests to be passed